packages/stage-pages/src/pages/settings/modules/speech.vue

```diff
@@ -10,6 +10,7 @@ import {
   TestDummyMarker,
   VoiceCardManySelect,
 } from '@proj-airi/stage-ui/components'
+import { useAnalytics } from '@proj-airi/stage-ui/composables/use-analytics'
```

Comment:
```suggestion
import { useAnalytics } from '@proj-airi/stage-ui/composables'
```

After exported, you can import them directly.